spec/acceptance/rules_spec.rb

```diff
@@ -257,6 +265,7 @@ class { '::firewall': }
         /-A INPUT -s 10.0.0.0\/(8|255\.0\.0\.0) -p icmp -m comment --comment \"013 icmp echo-request\" -m icmp --icmp-type 8 -j ACCEPT/,
         /-A INPUT -p icmp -m comment --comment \"013 icmp time-exceeded\" -m icmp --icmp-type 11 -j ACCEPT/,
         /-A INPUT -p tcp -m multiport --dports 22 -m comment --comment \"020 ssh\" -m state --state NEW -j ACCEPT/,
+        /-A INPUT -p tcp -m multiport --dports 22 -m comment --comment \"22 ssh allow\" -j ACCEPT/,
```

Comment:
Looking at the line above, that already seems to try to address this, but - judging from the problems we have - fail to do so. Then it can be removed to avoid carrying additional baggage.

Also, it might point to the fact that resetting the FW confuses the netfilter conntrack state that is used by `-m state` being closer to the root cause of all of these issues?